src/main/java/core/basesyntax/servises/Mapper.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.servises;
+
+import core.basesyntax.model.TransferDto;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Mapper {
+    private static final int TYPE = 0;
+    private static final int NAME = 1;
+    private static final int QUANTITY = 2;
+    private static final String SEPARATOR = ",";
+    private Validator validator = new ValidatorImpl();
```

Comment:
You do not have this interface/class.
Create your own Validator interface and implement the `validate()` method in it